docs/source/python/api/compute.rst

```diff
@@ -498,3 +498,50 @@ Structural Transforms
    make_struct
    replace_with_mask
    struct_field
+
+Compute Options
+---------------
+
+.. autosummary::
+   :toctree: ../generated/
+
+   ArraySortOptions
+   AssumeTimezoneOptions
```

Comment:
I am not sure it has much value to list them here, as long as they have no docstring.. (this will create a lot new doc pages, which will basically be empty)

Reply:
I list them here because it provides the signature and thus which arguments they support. As you said there won't be any docstring but given that in many cases you can guess what the arguments do from their name it's better than nothing

Reply:
Yeah, it's true that the signatures show something. It only looks kind of "bad" to have an empty docstring page ..

Actually, what happens if you leave out the `:toctree: ../generated/` (to only have the table) ? Although that will make just sphinx complain about nonexisting references ..

Reply:
Removing it also removes the reference page where you can see the signature

Reply:
> Removing it also removes the reference page where you can see the signature

The table doesn't show the signature?

Reply:
Nope